### PR TITLE
Readme typos, clarity. Better directus executor failing

### PR DESCRIPTION
**Pull request by tj-boyle**
Fix of various typos and missing clarity of READMEs.

Added an additional step to CMS setup regarding creating a DB schema that matches the `DB_SEARCH_PATH` environment variable. The env example has `cms` as a value there, so if you keep that value without creating a schema, `nx bootstrap cms` will fail.

BUT it will tell you it succeeded - so I additionally modified the executor to throw an error if there's an error instead of always returning success.

May also modify bottom portion of the CMS Readme before merge, see: https://rocketinsights.slack.com/archives/C01RTKC55R6/p1643668237913089

### CODE REVIEW

File: tools/executors/directus/impl.js [hunk 1]
```diff
@@ -174,7 +174,10 @@ function runAction(options, context) {
           directus.on('error', function (error) {
             reject(error)
           })
-          directus.on('exit', function () {
+          directus.on('exit', function (code) {
+            if (code !== 0) {
+              reject(code)
```
Comment (smonn) [inline tools/executors/directus/impl.js:179]: I _think_ we need a `return` statement here too.

```suggestion
              reject(code)
              return
```

File: tools/executors/directus/impl.ts [hunk 2]
```diff
@@ -45,7 +45,11 @@ async function runAction(
       reject(error)
     })
 
-    directus.on('exit', () => {
+    directus.on('exit', (code) => {
+      if (code !== 0) {
+        reject(code)
```
Comment (smonn) [inline tools/executors/directus/impl.ts:50]: Same as above.

```suggestion
        reject(code)
        return
```

**Review by smonn — APPROVED** [review]
Thanks for this!

File: tools/executors/directus/impl.js (hunk 1 above)
Comment (tj-boyle) [reply tools/executors/directus/impl.js:179]: yep turns out. the promise stays rejected but it'd still execute any other code afterward if not returned, never really even considered that!

Comment (smonn) [reply tools/executors/directus/impl.js:179]: Now that I think about it, another option is to wrap the resolve in an else block.

**Review by smonn — APPROVED** [review]

**Pull request merged**